README.md

```diff
@@ -2157,7 +2157,7 @@ In order to achieve greater coverage and encourage more people to contribute to
         <tr>
             <td><a href="https://en.wikipedia.org/wiki/Linked_list">Doubly Linked List</a></td>
             <td> <!-- C -->
-                <a href="./src/c/ListaDuplamenteEncadeada.c">
+                <a href="./src/c/DoublyLinkedList.c">
```

Comment:
This link is broken, could you please check if the file was actually renamed?

Reply:
solved in the lasts commits